tox.ini

```diff
@@ -80,21 +80,33 @@ deps =
 setenv =
   TEST_JUJU_CHANNEL = 2.9/stable
 changedir = {toxinidir}/tests/functional
-commands = functest-run-suite {posargs:--keep-faulty-model}
+commands = functest-run-suite
```

Comment:
We should use `--keep-faulty-model` to be able to run generate juju-crashdump [here](https://github.com/charmed-kubernetes/actions-operator/blob/main/src/cleanup/index.ts#L53).

I also do not understand of need for multiple tox environmnet, since we can achieve same behavior with.
`tox -e func` -> `TEST_JUJU_CHANNEL = 2.9/stable tox -e func`
`tox -e func33-jammy` -> `TEST_JUJU_CHANNEL = 3.3/stable tox -e func -- -b jammy --keep-faulty-model`
`tox -e func33-foca` -> `TEST_JUJU_CHANNEL = 3.3/stable tox -e func -- -b focal --keep-faulty-model`

Doing it like this, we do not need to change anything in Makefile.